packs/equipment/drake-rifle-acid.json

```diff
@@ -51,15 +51,24 @@
         "reload": {
             "value": "1"
         },
+        "requireAmmoToFire": false,
         "rules": [
             {
                 "key": "Note",
                 "outcome": [
                     "criticalSuccess"
                 ],
                 "selector": "{item|_id}-damage",
-                "text": "On a critical hit, the spittle clings to the target and they take persistent damage of the same type as the weapon equal to @Damage[(1d4 + @item.system.damage.dice)[persistent,acid]]{1d4 + the number of weapon damage dice}. A drake rifle does not add critical specialization effects.",
+                "text": "On a critical hit, the spittle clings to the target and they take persistent damage of the same type as the weapon equal to @Damage[(1d4 + @item.system.damage.dice)[persistent,@item.system.damage.damageType]]{1d4 + the number of weapon damage dice}.",
```

Comment:
We need to localize all of this text by adding it into re-en.json and calling that here. We should also just make this the crit spec and then grant crit spec with just this one weapon for anyone wielding one.